packages/generator/test/template-helpers-format-for-tsdoc.test.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+
+import { expect } from "chai";
+
+import { formatForTsDoc } from "../src/templateHelpers";
+
+describe("Test formatForTsDoc template help function", () => {
```

Comment:
The first half of these tests are `.to.eq()` and the second half are `.to.be.eq()` :|

Reply:
good catch